suricata.yaml.in

```diff
@@ -155,6 +155,8 @@ outputs:
         - files:
             force-magic: no   # force logging magic on all logged files
             force-md5: no     # force logging of md5 checksums
```

Comment:
I just wanted to let you know that your reasoning makes sense, however, I won't be able to implement it in the next two weeks due to time constraints, most of the time I will be AFK. If the powers that be (looking at you @inliniac :P) decide this is worth to be implemented, I will do it if no one picks it up (whether it is on a new version of this PR, or a new one).